src/components/GetApi/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from "react";
+import axios from "axios";
+
+const Index: React.FC = () => {
+  useEffect(() => {
+    weatherData();
+  }, []);
```

Comment:
specify dependencies so that the effect runs on every render

Reply:
Okay